src/Text/Taggy/Combinators.hs

```diff
@@ -11,7 +11,6 @@
 module Text.Taggy.Combinators (hasName, hasAttr, getAttr, innerText, (//), (/&), (/*), trees, subtrees) where
 
 import Prelude hiding (lookup)
-import Data.Monoid (mconcat)
```

Comment:
Oh, this is in the Prelude as of 7.10. Does `base` still provide it via `Monoid`? If not, maybe an `ifdef` on base?